task/aontimer/src/main.rs

```diff
@@ -0,0 +1,82 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#![no_std]
+#![no_main]
+
+// NOTE: you will probably want to remove this when you write your actual code;
+// we need to import userlib to get this to compile, but it throws a warning
+// because we're not actually using it yet!
```

Comment:
Comment likely not needed anymore

Reply:
yep, removed the import but missed the comment.